src/test/ui/resolve/issue-21221-1.stderr

```diff
@@ -16,10 +18,11 @@ error[E0412]: cannot find type `Mul` in this scope
    |                ^^^ not found in this scope
    |
 help: possible candidates are found in other modules, you can import them into scope
-   | use mul1::Mul;
-   | use mul2::Mul;
-   | use mul3::Mul;
-   | use mul4::Mul;
+   |
+11 | use mul1::Mul;
+11 | use mul2::Mul;
+11 | use mul3::Mul;
+11 | use mul4::Mul;
```

Comment:
this looks a bit surprising to me -- I wonder if we want to disable line numbers sometimes? or..maybe it's ok. Kind of helps convey that these are distinct alternatives.